cas-server-support-oauth/src/main/java/org/jasig/cas/support/oauth/validator/OAuthValidator.java

```diff
@@ -0,0 +1,111 @@
+package org.jasig.cas.support.oauth.validator;
+
+import org.apache.commons.lang3.StringUtils;
+import org.jasig.cas.services.ServicesManager;
+import org.jasig.cas.support.oauth.OAuthConstants;
+import org.jasig.cas.support.oauth.services.OAuthRegisteredService;
+import org.jasig.cas.support.oauth.util.OAuthUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.validation.constraints.NotNull;
+
+/**
+ * Validate OAuth inputs.
+ *
+ * @author Jerome Leleu
+ * @since 4.3.0
+ */
+@Component("oAuthValidator")
+public class OAuthValidator {
+
+    /** The logger. */
+    protected final Logger logger = LoggerFactory.getLogger(getClass());
+
+    /** The services manager. */
+    @NotNull
+    @Autowired
+    @Qualifier("servicesManager")
+    protected ServicesManager servicesManager;
+
+    /**
+     * Check if a parameter exists.
+     *
+     * @param request the HTTP request
+     * @param name the parameter name
+     * @return whether the parameter exists
+     */
+    public boolean checkParameterExist(final HttpServletRequest request, final String name) {
+        final String parameter = request.getParameter(name);
```

Comment:
I haven't found any method doing that in Spring: can you elaborate a little more? Are you referring to `@RequestParam`?